static/docs/command-reference/pipeline/show.md

```diff
@@ -52,6 +51,29 @@ instead of stages.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Paging the output
+
+This command's output is automatically pushed to `less` command line tool if
+`less` is runnable (installed and can be executed in your shell). To be exact,
+the actual command is `less --chop-long-lines --clear-screen`.
```

Comment:
just wondering if we tested this with GNU as well as BSD less and other popular versions? cc @efiop

Reply:
BSD less is the one on mac? If so - yes 🙂

Reply:
then resolving it..

Reply:
👍